logger/logger.go

```diff
@@ -72,12 +79,7 @@ type Logger struct {
 }
 
 // SetConfig allows configuration changes in one go
-func (l *Logger) SetConfig(source string, logLvl LogLevel, formatter Formatter, output io.Writer) {
-	if l.globals == nil {
-		l.globals = make(map[string]interface{})
-	}
-
-	l.globals["source"] = source
+func (l *Logger) SetConfig(logLvl LogLevel, formatter Formatter, output io.Writer) {
```

Comment:
What's the benefit of this change? If we can add this context logger without breaking the API, we should unless there's a large reason to break it.